app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -127,174 +125,179 @@ private fun DetailsContent(
                 .padding(internalPadding)
                 .fillMaxHeight()
         ) {
-            Column(
+            val listState = rememberLazyListState()
+            val keyboardOptions = KeyboardOptions(KeyboardCapitalization.Words, true, KeyboardType.Text, ImeAction.Next)
+            val keyboardController = LocalSoftwareKeyboardController.current
+            val focusRequesterFirstName = remember { FocusRequester() }
+            val focusRequesterLastName = remember { FocusRequester() }
+            val focusRequesterTeamName = remember { FocusRequester() }
+
+            LazyColumn(
+                state = listState,
                 horizontalAlignment = Alignment.CenterHorizontally,
                 verticalArrangement = Arrangement.Top,
                 modifier = Modifier
-                    .weight(weight = 1f, fill = true)
-                    .verticalScroll(scrollState)
+                    .weight(weight = 1f)
+                    .imeNestedScroll()
```

Comment:
I think it makes the scroll to behave very strangely:

https://github.com/wireapp/wire-android-reloaded/assets/30429749/564702af-dcb8-4bd3-ae05-4be12620ee59

Reply:
Removed